crates/ide_assists/src/handlers/inline_function.rs

```diff
@@ -2,14 +2,64 @@ use ast::make;
 use hir::{HasSource, PathResolution};
 use syntax::{
     ast::{self, edit::AstNodeEdit, ArgListOwner},
-    AstNode,
+    ted, AstNode,
 };
 
 use crate::{
     assist_context::{AssistContext, Assists},
     AssistId, AssistKind,
 };
 
+// Assist: inline_method
```

Comment:
From the user’s perspective, I think there should be only one assist: “inline call”, which works for function and method calls.

And some day we’d want to create “inline function” that is calllable at the der-site, and inlines all usages.